canonical-web-and-design/ubuntu.com
# Observability build

## Done
Building /observability/what-is-observability per copydoc: https://docs.google.com/document/d/1eoqDQzFiRJY2u_EO4JBuLBLqae4K3nrEf3qd88KHw0Y/edit

zeplin:
https://app.zeplin.io/project/6143075beaee62250b828788/screen/614307aa6a68af5099c01a44 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-10439.demos.haus/observability/what-is-observability
- check against copy and zeplin mock-up (see above)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4380

## Screenshots
![Screenshot 2021-09-21 at 18 03 39](https://user-images.githubusercontent.com/17607612/134206407-db101d87-8018-44ec-a7ff-1611701c759b.png)

![Screenshot 2021-09-21 at 18 03 59](https://user-images.githubusercontent.com/17607612/134206449-9ee30e37-d306-4ff4-a462-8e53744ed56e.png)
![Screenshot 2021-09-21 at 18 04 13](https://user-images.githubusercontent.com/17607612/134206490-85c8438b-8dea-4cac-b2e3-35e53053172a.png)
